foursight_core/app_utils.py

```diff
@@ -616,7 +1226,7 @@ def process_view_result(self, connection, res, is_admin):
                 del res['action']
         return res
 
-    def view_foursight_history(self, environ, check, start=0, limit=25, is_admin=False,
+    def view_foursight_history(self, request, environ, check, start=0, limit=25, is_admin=False,
```

Comment:
This is an incompatible change. Is it only local callers that will care?  In general, the safest thing when unsure is to add new args at the rightmost end so that positional calls are not disturbed. This is also why I tend to put `*, ` in arglists once you start to get to more than one or two arguments because really the human brain deals poorly with randomly arranged positional args unless they map neatly to a well-known verbs, and most verbs in english take only a direct and indirect object (so 1 or 2 args) positionally. The rest they pass by keyword (preposition). keyword arguments are not ordered.

Reply:
Noted, thanks. I believe I've got all callers of this covered (foursight-cgap, foursight). Will keep this in mind.